backend/models/type_operation.go

```diff
@@ -0,0 +1,35 @@
+package models
+
+type OperationType string
+
+const (
+	Create OperationType = "create"
+	Update OperationType = "update"
+	Read   OperationType = "read"
+	Delete OperationType = "delete"
+	List   OperationType = "list"
+	All    OperationType = "*"
+	all    string        = "*"
```

Comment:
Is it necessary?

Reply:
What is necessary? the last 'all' definition? I guess no. Or do u mean entire OperationType?

Reply:
@samuelus I am living this in as I am not sure what you refer to